tests/aws/s3/test_s3.py

```diff
@@ -5982,6 +5982,7 @@ def test_post_request_missing_signature(
         )
 
         # AWS seems to detected what kind of signature is missing from the policy fields
+        print(response.content)
```

Comment:
will remove in the next PR if possible, as I wouldn't want to rebase everything. It could be merged right after!